website/docs/sidebar.md

```diff
@@ -0,0 +1,303 @@
+---
+id: docs-sidebar
+title: Sidebar
+slug: /sidebar
+---
+
+To generate a sidebar to your Docusaurus site:
+
+1. Define a file that exports a [sidebar object](#sidebar-object).
+1. Pass this object into the `@docusaurus/plugin-docs` plugin directly or via `@docusaurus/preset-classic`.
+
+```js {8-9} title="docusaurus.config.js"
+module.exports = {
+  // ...
+  presets: [
+    [
+      '@docusaurus/preset-classic',
+      {
+        docs: {
+          // Sidebars filepath relative to the site dir.
+          sidebarPath: require.resolve('./sidebars.js'),
+        },
+        // ...
+      },
+    ],
+  ],
+};
+```
+
+## Sidebar object
+
+A sidebar object contains [sidebar items](#understanding-sidebar-items) and it is defined like this:
+
+```typescript
+type Sidebar = {
+  [sidebarId: string]:
+    | {
+        [sidebarCategory: string]: SidebarItem[];
+      }
+    | SidebarItem[];
+};
+```
+
+For example:
+
+```js title="sidebars.js"
+module.exports = {
+  docs: [
+    {
+      type: 'category',
+      label: 'Getting Started',
+      items: ['greeting'],
+    },
+    {
+      type: 'category',
+      label: 'Docusaurus',
+      items: ['doc1'],
+    },
+  ],
+};
+```
+
+In this example, notice the following:
+
+- The key `docs` is the id of the sidebar. The id can be any value, not necessarily `docs`.
+- `Getting Started` is a category within the sidebar.
+- `greeting` and `doc1` are both [sidebar item](#sidebar-item).
+
+Shorthand notation can also be used:
+
+```js title="sidebars.js"
+module.exports = {
+  docs: {
+    'Getting started': ['greeting'],
+    Docusaurus: ['doc1'],
+  },
+};
+```
+
+:::note Shorthand notation relies on the iteration order of JavaScript object keys for the category name. When using this notation, keep in mind that EcmaScript does not guarantee `Object.keys({a,b}) === ['a','b']`, yet this is generally true. :::
+
+## Using multiple sidebars
+
+You can have multiple sidebars for different Markdown files by adding more top-level keys to the exported object.
+
+Example:
+
+```js title="sidebars.js"
+module.exports = {
+  firstSidebar: {
+    'Category A': ['doc1'],
+  },
+  secondSidebar: {
+    'Category A': ['doc2'],
+    'Category B': ['doc3'],
+  },
+};
+```
+
+By default, the doc page the user is reading will display the sidebar that it is part of. This can be customized with the [sidebar type](#understanding-sidebar-items).
+
+For example, with the above example, when displaying the `doc2` page, the sidebar will contain the items of `secondSidebar` only. Another example of multiple sidebars is the `API` and `Docs` sections on the Docusaurus website.
+
+For more information about sidebars and how they relate to doc pages, see [Navbar doc link](docs/api/themes/configuration/#navbar-doc-link).
```

Comment:
```suggestion
For more information about sidebars and how they relate to doc pages, see [Navbar doc link](./api/themes/theme-configuration.md#navbar-doc-link).
```

Netlify CI detects a broken link here.

![image](https://user-images.githubusercontent.com/749374/101799167-21581880-3b0c-11eb-8b98-c8a2c204c9f9.png)

For linking from one doc to another inside a given version, it's better to use relative paths to the md file.